### PR TITLE
Nb data ingestion upcate invoice item query

### PR DESCRIPTION
**Pull request by NathanielBennett**
Adds unexported columns for InvoiceItem for data-ingestion project

### CODE REVIEW

File: handlers/zuora-datalake-export/src/main/scala/com/gu/zuora/datalake/export/ExportLambda.scala [hunk 1]
```diff
@@ -210,7 +210,7 @@ object Query extends Enum[Query] {
...
+    "SELECT accountingCode, appliedToInvoiceItemId, chargeAmount, chargeDate, chargeName, createdById, createdDate, invoice.Id, processingType, product.Description, product.Id, product.Name, productRatePlanCharge.Id, quantity, ratePlanCharge.Id, revRecStartDate, serviceEndDate, serviceStartDate, sku, subscriptionId, taxAmount, taxCode, taxExemptAmount, taxMode, unitPrice, uom, updatedById, updatedDate, id, account.Id, balance, accountingPeriod.Id, amendment.Id, billToContact.Id, billToContactSnapShot.Id, defaultPaymentMethod.Id, deferredRevenueAccountingCode.Id, quantity, revRecStartDate, serviceEndDate, serviceStartDate, sku, invoice.Id, journalEntry.Id, journalRun.Id, parentAccount.Id, product.Id, productRatePlan.Id, productRatePlanCharge.Id, ratePlan.Id, ratePlanCharge.Id, soldToContact.Id, soldToContact.Id, subscription.Id FROM InvoiceItem  WHERE (CreatedDate >= '2019-08-28T00:00:00') AND (CreatedDate <= '2099-01-01T00:00:00')",
```
Comment (paulbrown1982) [inline handlers/zuora-datalake-export/src/main/scala/com/gu/zuora/datalake/export/ExportLambda.scala]: @NathanielBennett there are some repetitions in here

Comment (NathanielBennett) [reply handlers/zuora-datalake-export/src/main/scala/com/gu/zuora/datalake/export/ExportLambda.scala]: i **THINK** i've got them all

**Comment by mario-galic** [comment]
Remember to adjust the dates in the filter after full export as per FIXME:

```
//  FIXME: Zuora is not capable of doing a full export of InvoiceItem hence the WHERE clause: https://support.zuora.com/hc/en-us/requests/186104
```

**Comment by mario-galic** [comment]
@NathanielBennett https://github.com/guardian/support-service-lambdas/tree/master/handlers/zuora-datalake-export#how-to-extract-a-full-load-of-invoiceitem

File: handlers/zuora-datalake-export/src/main/scala/com/gu/zuora/datalake/export/ExportLambda.scala [hunk 2]
```diff
@@ -115,7 +115,7 @@ object Postconditions extends (JobResults => Either[Throwable, String]) with Laz
            |  Name: ${batch.name}
            |  Change size: ${batch.recordCount}
            |  Changes since: ${jobResult.incrementalTime}
-           |  Target S3 bucket: ${Query.withName(batch.name).targetBucket}
+           |  Target S3 bucket: ${Query.withName(batc0h.name).targetBucket}
```
Comment (mario-galic) [inline handlers/zuora-datalake-export/src/main/scala/com/gu/zuora/datalake/export/ExportLambda.scala]: Typo `batc0h` -> `batch`

**Pull request merged**